py-rse/packaging.Rmd

```diff
@@ -1305,6 +1305,41 @@ and how well it is documented,
 so the peer review process can be a great way to get critical feedback
 from people who have seen many research software packages come and go over the years.
 
+Once you have obtained a DOI and possibly published with a software journal,
+the last step is to tell users how to cite your new software package.
+This is traditionally done by adding a `CITATION` file to the associated GitHub repository
+(alongside `README`, `LICENSE`, `CONDUCT`, etc; \@ref(intro-boilerplate)),
```

Comment:
"alongside `README`, `LICENSE`, and similar files \@ref(intro-boilerplate)" - having "etc" inside parens seems odd to me.